app/uk/gov/hmrc/gform/sharedmodel/formtemplate/ComponentType.scala

```diff
@@ -485,11 +485,26 @@ object MiniSummaryRow {
   implicit val format: Format[MiniSummaryRow] = derived.oformat()
 }
 
-case class MiniSummaryList(rows: List[MiniSummaryRow]) extends ComponentType
+case class MiniSummaryList(rows: List[MiniSummaryRow], displayInSummary: DisplayInSummary = IsNotDisplayInSummary)
```

Comment:
Let's not use default value `= IsNotDisplayInSummary`, it is not needed

Reply:
Updated